jbpm-examples/src/main/java/org/jbpm/examples/multipleinstance/MultipleInstanceExample.java

```diff
@@ -32,6 +31,7 @@
 import org.kie.api.runtime.manager.RuntimeManagerFactory;
 import org.kie.api.task.TaskService;
 import org.kie.api.task.model.TaskSummary;
+import org.kie.test.util.db.PersistenceUtil;
```

Comment:
There is also PersistenceUtil class in jbpm, though you would probably have to declare dependency:

    <dependency>
      <groupId>org.jbpm</groupId>
      <artifactId>jbpm-persistence-jpa</artifactId>
      <type>test-jar</type>
      <scope>test</scope>
    </dependency>

So it is up to you.

Reply:
wouldn't that be spreading the logic ? I think it makes more sense to use test utils

Reply:
we already use this org.jbpm.persistence.util.PersistenceUtil in all jbpm unit tests. But given that jbpm-examples are de facto not tests, maybe it is better to stay with org.kie.test.util.db.PersistenceUtil.